classes/class.ilObjAdobeConnect.php

```diff
@@ -1765,4 +1777,20 @@ public function setAcLanguage($ac_language)
 	{
 		$this->ac_language = strtolower($ac_language);
 	}
+
+	/**
+	 * @return bool
+	 */
+	public function getHtmlClient()
```

Comment:
Please change the name of accessor and mutator (same as in the plugin administration).